<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color:white;
}

QLabel{
	font-family: Arial;
	font-size: 14px;
}

QPushButton{
	background-color: #ed5684;
	color: white;
	min-height:30px;
	border-radius: 20px;
}

QLineEdit{
	border: 1px solid #d0d0d0;
	border-radius: 20px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item alignment="Qt::AlignVCenter">
    <widget class="QFrame" name="asdasd">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:26px;
color: #ed5684;</string>
        </property>
        <property name="text">
         <string>Реєстрація</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QFrame" name="verticalFrame_4">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item alignment="Qt::AlignLeft">
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Номер телефону</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="email_line">
             <property name="styleSheet">
              <string notr="true">min-height:30px;</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Імя</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="name_line">
             <property name="styleSheet">
              <string notr="true">min-height:30px;</string>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Пароль</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="password_line">
             <property name="contextMenuPolicy">
              <enum>Qt::NoContextMenu</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">min-height:30px;</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="asdasda">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="login_redirect_button">
           <property name="text">
            <string>Перейти до входу</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="register_button">
           <property name="text">
            <string>Зареєструватися</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
